meteor-app/imports/ui/assessment/assessment-add-review.js

```diff
@@ -22,19 +32,29 @@ const AssessmentAddReview = (props) => {
               </h2>
               <Segment.Group>
                 {
+                  // TODO: Need to add estimated price quote
                   Object.keys(step.schema.properties).map((key, value, iy) => {
-                    if (props.formData[key]) {
+                    if (props.formData[key] && key !== 'serviceCost' && key !== 'partsCost') {
                       return (
-                        <Segment key={iy}>
+                        <Segment key={key}>
                           <strong>{step.schema.properties[key].title}</strong>
-                          <span style={{ paddingLeft: '1em' }}>
-                    
+                          <ul style={{ paddingLeft: '1em' }}>
                             {
-                              <span>
-                                {props.formData[key]}
-                              </span>
+                              Array.isArray(props.formData[key]) ?
+                              props.formData[key].map(item => {
+                                return (
+                                  <li key={item}>{item}</li>
+                                )
+                              }) :
+                              <li>
+                                {
+                                  typeof props.formData[key] === 'boolean' ?
```

Comment:
Multiple ternary expressions on a line are deprecated. It is hard to read and debug. It also makes the rendering markup complicated.
I would have a block of code before the return statement that assembles the data into an array of items, and the rendering code simply iterates over the array. The end result is much cleaner and therefore easier to troubleshoot.